### PR TITLE
check that dynamic conf was downloaded before proceeding

### PR DESCRIPTION
**Pull request by gravitypriest**

### CODE REVIEW

File: insights/client/collection_rules.py [hunk 1]
```diff
@@ -99,20 +100,25 @@ def get_collection_rules(self, raw=False):
...
-            logger.error("Debug Info: \nConf status: %s", req.status_code)
-            logger.error("Debug Info: \nConf message: %s", req.text)
-            return False
+                json_response = NamedTemporaryFile()
+                json_response.write(req.text)
+                json_response.file.flush()
+            else:
+                logger.error("ERROR: Could not download dynamic configuration")
+                logger.error("Debug Info: \nConf status: %s", e.status_code)
+                logger.error("Debug Info: \nConf message: %s", e.text)
```
Comment (kylape) [inline insights/client/collection_rules.py]: On these two debug lines, did you mean `req.status_code` and `req.text` instead of `e`?

**Comment by kylape** [comment]
This resolves https://github.com/RedHatInsights/insights-core/issues/695

**Comment by gravitypriest** [comment]
Yup, small typo there. Will fix

File: insights/client/collection_rules.py (hunk 1 above)
Comment (gravitypriest) [reply insights/client/collection_rules.py]: Yup, small typo there. Will fix

Comment (gravitypriest) [reply insights/client/collection_rules.py]: fixed~

**Pull request merged**